classes/fields.class.php

```diff
@@ -211,7 +211,7 @@ function render_field_settings() {
 		}
 
 		$html .= '</div>';
-		echo $html;
+		echo ppom_esc_html( $html );
 	}
```

Comment:
we should not rely on built-in `ppom_esc_html`

if some elements are not necessarily; we can optimize/fix `ppom_esc_html` or have a new context for `wp_kses_allowed_html`

thats valid all codebase which `ppom_esc_html` is used.

**an example output:**
![Screen Shot 2023-04-26 at 14 22 35](https://user-images.githubusercontent.com/25361626/234560431-85807b43-ef0b-44c2-b41a-7d66302079da.jpg)

Reply:
I would like to confirm whether it is necessary to allow scripts and other elements. Could you please check and share your findings with me? Once we have the results, we can move forward.

Reply:
@ mskapusuz I think we are not using script and style tags. Should I remove it from the `ppom_esc_html` function?
Ref: https://github.com/Codeinwp/woocommerce-product-addon/blob/master/inc/validation.php#L46

Reply:
Today in later hours I'll continue to my reviews

Reply:
@mskapusuz Removed script/style from the `ppom_esc_html` function